From e2aafc77176658fdd5b8338e8f3f0bbe68013d64 Mon Sep 17 00:00:00 2001
From: ran shalit <rshalit@habana.ai>
Date: Mon, 13 Sep 2021 12:22:39 +0300
Subject: [PATCH] [SW-0] gaudi: remove drain init

Drain init for gaudi is disabled by default in HW.
Therefore, no need to disable it again in SW.
Not only this, but the init function was never
called anyway. Therefore, drain init is removed.

Change-Id: I4861dd0e159c9c8f2d45920131b3a668ce2d1495
Signed-off-by: ran shalit <rshalit@habana.ai>
---

diff --git a/arch/gaudi/arm/asic.c b/arch/gaudi/arm/asic.c
index a9c72a1..9c757de 100644
--- a/arch/gaudi/arm/asic.c
+++ b/arch/gaudi/arm/asic.c
@@ -180,12 +180,6 @@
 	return 0;
 }
 
-void hl_pcie_drain_init(void)
-{
-	regmap_writel(0, mmPCIE_WRAP_LBW_DRAIN_CFG);
-	regmap_writel(0, mmPCIE_WRAP_HBW_DRAIN_CFG);
-}
-
 #define BOOTSTRAP_JTAG_EN BIT(12)
 
 bool hl_is_jtag_enabled(void)
